templates/operations.ts.hbs

```diff
@@ -145,6 +145,11 @@
         {{/if}}
       {{/each}}
 
+      const headers = (options && options.headers) ? { ...options.headers } : {};
```

Comment:
Can this be a helper function so that we can test?

Reply:
This is just a getter with a default value. Not much to really test here.